capstone/src/main/java/com/google/univiz/scorecard/CollegeData.java

```diff
@@ -0,0 +1,31 @@
+package com.google.univiz.scorecard;
+
+/**
+ * CollegeData is an interface that contains methods and functions which communicate with the
+ * CollegeScorecardAPI to obtain college data from the URL in JSON format and parse them to become
+ * ScorecardData java objects.
+ */
+public interface CollegeData {
+
+  /** TODO(biancamacias): decide what getSuggestions does */
+  public List<ScorecardData> getSuggestions(String suggestion) {}
+
+  /**
+   * Returns a list of ScorecardData to be used by the servlets.
+   *
+   * @param ids A list of college IDs that were selected by the user.
+   * @param fieldNames A list of fieldNames of data requested that will be used to call the
+   *     CollegeScorecard API?
+   * @return A list of ScorecardData types to be used in the back-end.
+   */
+  public List<ScorecardData> getCollegesById(List<String> ids, List<String> fieldNames) {}
```

Comment:
In [Kaveh's layout](https://docs.google.com/document/d/1G2QrgDMhVCPX6cbNOQNKvexDImB5Duw1dr50cDGx70U/edit?disco=AAAAKDmwaXI) he suggested a `@Nullable String sortByField` parameter for this function, might be useful if we want to deal with data based on how it's sorted under the specified parameter. I don't know if it's necessary for the function of this though, but it's probably worth consideration.